### ISSUE TITLE
SPPA - Create private page content type

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#3922]
# Motivation

SPPA has a career services manual that was previously only available to logged-in students. That page is not publicly visible. https://sppa.prod.drupal.uiowa.edu/student-resources/career-services-manual

# Story

As an SPPA site owner I want to be able to create page content that is only viewable by logged-in users.

# Proposed solution

- Add a "Private page" content type that is like the standard page, but has viewing privileges restricted to authenticated users. 
- Add this config to SPPA site split.

**Comment by joewhitsitt** [comment]
>that is like the standard page

core page or our layout builder page?

**Comment by pyrello** [comment]
Apologies for the vague description. We had discussed just using a body field and not including LB and I failed to capture that detail. I don't think there is a problem with including LB other than that it requires that we keep the private page LB config up-to-date with changes to the page config.

Ideally, I think we should replace this with something that allows a regular page to be marked as "private".

Also, we changed the name of the page from "Basic page" to just "Page" because a page with a layout tool is anything but basic ;)
